README.md

```diff
@@ -78,6 +85,12 @@ alias zq="zoxide query"
 alias zr="zoxide remove"
 ```
 
+#### fish
+Using [fisher](https://github.com/jorgebucaran/fisher):
```

Comment:
Can we make this generic to all fish plugin managers?

Reply:
I'll also add support for `oh-my-fish`